docs/api/QUIC_REGISTRATION_CONFIG.md

```diff
@@ -0,0 +1,34 @@
+QUIC_REGISTRATION_CONFIG structure
+======
+
+The structure used to configure the execution context of a application registration.
+
+# Syntax
+
+```C
+typedef struct QUIC_REGISTRATION_CONFIG {
+    const char* AppName;
+    QUIC_EXECUTION_PROFILE ExecutionProfile;
+} QUIC_REGISTRATION_CONFIG;
+```
+
+# Members
+
+`AppName`
+
+An optional (may be `NULL`), null-terminated string describing the application that created the registration. This field is primarily used for debugging purposes.
+
+`ExecutionProfile`
+
+Provides a hint to MsQuic on how to optimize its thread scheduling operations.
+
+**Value** | **Meaning**
+------ | ------
+**QUIC_EXECUTION_PROFILE_LOW_LATENCY**<br>0 | Indicates that scheduling should be generally optimized for reducing response latency. *The default execution profile.*
+**QUIC_EXECUTION_PROFILE_TYPE_MAX_THROUGHPUT**<br>1 | Indicates that scheduling should be optimized for maximum single connection throughput.
```

Comment:
We should add a note here that you should test if changing the profile is better for your use case, as in many cases LOW_LATENCY can actually result in much higher throughput then MAX_THROUGHPUT.

Reply:
I'll leave that for a future PR.